### DISCUSSION TITLE
Select component – issue with styles.

### DISCUSSION THREAD

**Discussion by martinsnajdr** [wireui/wireui#557] [BUG]
Hello,

I have a two small problems with the select component. 

1. The default selected value is loaded ok, but the styles (semibold font + check icon) that should be added to selected option are not added on init load. But when I actively select another option, the styles are added.
2. The popup sometimes closes automatically, so I have to click twice.

Here is a video:
https://www.dropbox.com/s/g0o8tjbxvmfacx2/bug-wireui.mov?dl=0]

**Comment by ph7jack** [comment]
@martinsnajdr Could you test the latest version?

**Reply by ph7jack** [reply]
Also, provide informations about your application.

**Comment by miguel24rueda** [comment]
@martinsnajdr hola, amigo como hizo para cambiarle los estilos al componente select ya no deja publicarlo
